bin/Powheg/run_pwg.py

```diff
@@ -1138,7 +1121,9 @@ def runhnnlo(folderName, njobs, QUEUE):
         prepareJob(tagName, '', '.')
 
         if not os.path.exists(args.inputTemplate) :
-            os.system('wget --quiet --no-check-certificate -N http://cms-project-generators.web.cern.ch/cms-project-generators/'+args.inputTemplate+' -O '+args.folderName+'/powheg.input')
+            m_ret = os.system('wget --quiet --no-check-certificate -N http://cms-project-generators.web.cern.ch/cms-project-generators/'+args.inputTemplate+' -O '+args.folderName+'/powheg.input')
```

Comment:
question: these lines make it such that when no input file neither passed nor present, a default version is downloaded, right? Would it make sense to prevent that no datacard argument is passed (i.e. raise an exception if this happens), and instead add another argument to explicitly use the default cards (which are there usually for testing purpose)?